tutorials/ragas_retrieval_evals_tutorial.ipynb

```diff
@@ -137,7 +137,9 @@
    "metadata": {},
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    assert openai_api_key != "", "❌ Please set your OpenAI API key"
</pre><p>Probably good to nix this line.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/1115/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
Leaving it just in case :) but yeah happy to remove